pkg/handler/handler.go

```diff
@@ -174,14 +174,22 @@ func (h handler) deprovision(w http.ResponseWriter, r *http.Request, params map[
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "invalid instance_uuid"})
 		return
 	}
+
 	var async bool
 	queryparams := r.URL.Query()
```

Comment:
So small nit, just because your editing this.

I would prefer to use the [request.FormValue()](https://golang.org/pkg/net/http/#Request.FormValue). I personally think it is nicer and easier to use.